test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -14,6 +14,10 @@ extern "C" {
 #include <ucp/tag/tag_match.h>
 }
 
+#define UCP_INSTANTIATE_TAG_OFFLOAD_TEST_CASE(_test_case) \
+    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, dcx, "dc_x") \
+    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, rcx, "rc_x")
```

Comment:
Do we have to do it through hardcoded  transport names or we can do it though runtime query for transport support for tag matching ?

Reply:
imo, either way is ok.
We do not have infra to select proper transports just once now, so we run every test with every transport, do runtime query and then skip if not supported. This results in longer execution time and flooding of test output.

Reply:
@shamisp, is it ok to merge?

Reply:
👍